pandas/plotting/_core.py

```diff
@@ -2505,7 +2507,7 @@ def line(self, **kwds):
 
         Parameters
         ----------
-        **kwds : optional
+        \*\`**kwds` optional
```

Comment:
Why is the additional `\*\` in the beginning needed? (compared to the ones above)